thorlcr/activities/diskread/thdiskreadslave.cpp

```diff
@@ -348,13 +349,13 @@ void CDiskRecordPartHandler::open()
                     else
                     {
                         remoteReadException.setown(e);
-                        remoteReadExceptionPath.set(path);
+                        remoteReadExceptionPath.set(filename);
```

Comment:
Curious why this (and line 358 below) changed.

Reply:
path.clear() fixed the tracing issue (comment from commit now in PR), the switch to 'filename' doesn't change anything (it is same as content of path), it just seemed better to use filename rather than the reused temporary 'path'